src/Microsoft.DotNet.Tools.Test/Program.cs

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.Extensions.Testing.Abstractions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    public class Program
+    {
+        public static int Main(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            var app = new CommandLineApplication(false)
+            {
+                Name = "dotnet test",
+                FullName = ".NET Test Driver",
+                Description = "Test Driver for the .NET Platform"
+            };
+
+            app.HelpOption("-?|-h|--help");
+
+            var parentProcessIdOption = app.Option("--parentProcessId", "Used by IDEs to specify their process ID. Test will exit if the parent process does.", CommandOptionType.SingleValue);
```

Comment:
Ah, fair enough. If there is already a "protocol" for these arguments in place, that makes sense. Can you put a small comment in the code to that effect to prevent "helpful" people from breaking it in the future?

Yeah... we tried Reviewable once on ASP.NET, but found it hard to get things in sync and most people don't bother clicking the "review on Reviewable" button :).